src/core/src/preprocess/pre_post_process.cpp

```diff
@@ -285,7 +285,7 @@ PreProcessSteps& PreProcessSteps::convert_element_type(const element::Type& type
 }
 
 PreProcessSteps& PreProcessSteps::resize(ResizeAlgorithm alg, size_t dst_height, size_t dst_width) {
-    OPENVINO_ASSERT(dst_height <= std::numeric_limits<int>::max() && dst_width <= std::numeric_limits<int>::max(),
+    OPENVINO_ASSERT(dst_height <= std::numeric_limits<size_t>::max() && dst_width <= std::numeric_limits<size_t>::max(),
```

Comment:
the code below casts dst_height  and dst_width. I suppose the intention of these asserts to make sure we don't truncate values